src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/controllers/v1/person/PLPController.kt

```diff
@@ -32,7 +33,7 @@ class PLPController(
 ) {
   @GetMapping("{hmppsId}/plp-induction-schedule")
   @Operation(
-    summary = "Returns plp the induction schedule associated with a person.",
+    summary = "Returns plp the current induction schedule associated with a person.",
```

Comment:
Minor, but on this and the next method, the `summary` tag reads weirdly.
Wonder if it should be something like:
`Returns the current PLP Induction Schedule associated with a person.`,
and
`Returns the PLP Induction Schedule history associated with a person.`

Reply:
ah yes - updated